packages/sanes-browser-extension/package.json

```diff
@@ -49,9 +49,9 @@
     "@iov/keycontrol": "^1.0.0-alpha.1",
     "@iov/lisk": "^1.0.0-alpha.1",
     "@iov/multichain": "^1.0.0-alpha.1",
-    "@material-ui/core": "^4.3.1",
-    "@material-ui/icons": "^4.2.1",
-    "@material-ui/styles": "^4.3.0",
+    "@material-ui/core": "^4.5.0",
+    "@material-ui/icons": "^4.4.3",
+    "@material-ui/styles": "^4.5.0",
```

Comment:
Can you rebase this onto master where those library upgrades should already be included?

Reply:
Unfortunately I forgot to include `sanes-browser-extension` `package.json` to PR with packages upgrade.